sdk/python/pulumi_eks/outputs.py

```diff
@@ -398,6 +402,15 @@ def __init__(__self__, *,
         :param str node_user_data_override: User specified code to run on node startup. This code is expected to handle the full AWS EKS bootstrapping code and signal node readiness to the managing CloudFormation stack. This code must be a complete and executable user data script in bash (Linux) or powershell (Windows).
                
                See for more details: https://docs.aws.amazon.com/eks/latest/userguide/worker.html
+        :param Sequence['NodeadmOptions'] nodeadm_extra_options: Extra nodeadm configuration sections to be added to the nodeadm user data. This can be shell scripts, nodeadm NodeConfig or any other user data compatible script. When configuring additional nodeadm NodeConfig sections, they'll be merged with the base settings the provider sets.
```

Comment:
This is already a generous doc comment but I would consider even more explicit, call out that these settings augment or override the base settings, and maybe give a quick motivating example (or a link to one).

Reply:
They do not necessarily overwrite. They can also be scripts for example. I can make this a bit clearer in the docs!